Kernel/Devices/Generic/RandomDevice.cpp

```diff
@@ -40,9 +40,12 @@ ErrorOr<size_t> RandomDevice::read(OpenFileDescription&, u64, UserOrKernelBuffer
     });
 }
 
-ErrorOr<size_t> RandomDevice::write(OpenFileDescription&, u64, UserOrKernelBuffer const&, size_t size)
+ErrorOr<size_t> RandomDevice::write(OpenFileDescription&, u64, UserOrKernelBuffer const& buffer, size_t size)
 {
-    // FIXME: Use input for entropy? I guess that could be a neat feature?
+    Bytes byte {};
+    TRY(buffer.read(&byte, size));
```

Comment:
What is the size of `Bytes byte {};` by default? what will happen if you pass `size > 1` then if it's only one byte?

Reply:
It's supposed to be indefinite but yeah this name doesn't make sense semantically